packages/desktop-ui-lib/src/lib/settings/plugins/services/builders/form-data.builder.ts

```
@@ -0,0 +1,120 @@
+			const versionData = this.buildVersionData(version);
+			this.appendFiltered(versionData, key);
+
+			if (!key) {
+				this.appendSource(version.sources, 'sources');
+			}
+		}
+		return this;
+	}
+
+	public appendSource(data: IPluginSource | IPluginSource[], key?: string, parentKey?: string): this {
+		key = parentKey ? `${parentKey}[${key}]` : key;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)